jetty-util/src/main/java/org/eclipse/jetty/util/thread/strategy/EatWhatYouKill.java

```diff
@@ -166,26 +159,25 @@ private void tryProduce(boolean wasPending)
             }
         }
 
-        final boolean non_blocking = Invocable.isNonBlockingInvocation();
-        
+        boolean nonBlocking = Invocable.isNonBlockingInvocation();
```

Comment:
note that the official jetty coding style says that local variables should not be camel cased, but use _.... but hey all you rebels long ago trashed any consistency to the coding style.... so yeehah cowboys!